README.md

```diff
@@ -357,23 +357,25 @@ It has more fleshed out support for some jobs but is *strongly* a Work In Progre
 
 |Job|Feature|
 |:-:|:-:|
-|<img src="./resources/ffxiv/jobs/pld-large.png" width="30px"/><br> Paladin|Shows current Oath amount, and atonement stacks. Also tracks Goring Blade DoT. |
-|<img src="./resources/ffxiv/jobs/war-large.png" width="30px"/><br> Warrior|Shows the beast amount, and tracks the remaining Storm's Eye buff time in gcds, and shows combo time remaining.|
+|<img src="./resources/ffxiv/jobs/pld-large.png" width="30px"/><br> Paladin|Shows Oath amount, and atonement stacks, DoT remaining time, Fight or Flight duration&cooldown, Expiacion cooldown and shows combo time remaining.|
+|<img src="./resources/ffxiv/jobs/war-large.png" width="30px"/><br> Warrior|Shows Beast amount, Surging Tempest duration, Upheaval&Inner Release cooldown, and shows combo time remaining.|
```

Comment:
Would you mind not combining words with `&`? I think it'd be easier to read if `&` was replaced with ` and ` everywhere, e.g. `Upheaval and Inner Release cooldown`.

Reply:
sure. I'll fix it after work